mozilla-release/mobile/android/base/java/org/mozilla/gecko/controlcenter/GlobalTrackersListAdapter.java

```diff
@@ -150,6 +153,7 @@ public void onClick(View v) {
         trackerCheckBox.setOnClickListener(new View.OnClickListener() {
             @Override
             public void onClick(View v) {
+                notifyDataSetChanged();
```

Comment:
why?

Reply:
so that no two list item show the options menu at the same time